src/v2/Components/NavBar/Menus/__tests__/MoreNavMenu.jest.tsx

```diff
@@ -9,6 +9,14 @@ jest.mock("v2/Artsy/Analytics/useTracking", () => ({
 }))
 
 describe("MoreNavMenu", () => {
+  beforeEach(() => {
+    jest.spyOn(console, "error").mockImplementation(() => {})
```

Comment:
Mind adding a FIXME comment above this and below describing the problem? This could mask future issues and would be hard to debug.